masterlist.yaml

```diff
@@ -6430,39 +6430,41 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 103
 
-  - name: 'Enhanced Vegetation \[\d{3}%\]\.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/23783/' ]
+  - name: 'Enhanced Vegetation( \[\d{3}%\])?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/23783/'
+        name: 'Enhanced Vegetation'
```

Comment:
The locations should be separated so that they don't apply to plugins that a location doesn't actually host:
```suggestion
```